types/index.test-d.ts

```diff
@@ -19,7 +19,7 @@ cheerio(html);
 cheerio('ul', html);
 cheerio('li', 'ul', html);
 
-const $fromElement = cheerio.load($('ul').get(0));
+const $fromElement = cheerio.load($('ul').toString());
```

Comment:
I believe idea here was test does "cheerio.load" accepts type "Element" as input, but I also did see error with last commit, so maybe cast it instead:
``` js
const $fromElement = cheerio.load($('ul').get(0) as Element);
```

I know type "Element" is not defined, but we could import it

``` js
import type { Element } from 'domhandler';
```

Reply:
That code is actually broken, even in JavaScript:

```js
const $fromElement = cheerio.load($('ul').get(0));
```

It throws an error. `cheerio.load()` doesn't accept an `Element`.

Reply:
Interesting, I tested it and tsd didn't complain. I also tested load function with  and it worked, like expected.

``` javascript
const cheerio = require('cheerio');
const html = 
  '<ul id="fruits">\n<li class="orange">Apple</li>\n<li class="class">Orange</li>\n<li class="pear">Pear</li>\n<input type="text" />\n</ul>';

const $ = cheerio.load(html);
const $fromElement = cheerio.load($('li').get(0));

console.log(cheerio.html($fromElement._root));
```
I copied what you did and added casting [cb6f40](https://github.com/cheeriojs/cheerio/commit/cb6f40f8607161c48405e975b03b79085e99aec6) and it works. Interesting why it fails on you.

Reply:
I'm not talking about tsd / TypeScript though - did you try actually running that as JavaScript? `cheerio.load()` with an element throws an error. The type for the `load` method also seems to indicate that it does not accept an element.

Reply:
Ah sorry I think I was looking at the wrong overload. It does seem to accept an element here: https://github.com/cheeriojs/cheerio/blob/1c2c0c0a86f60da50c03bbcd192b8e671178a3b6/types/index.d.ts#L282

Reply:
I've added the cast now, if you would prefer that over the `.html()` method.

Reply:
I have to admit I rarely use typescript, I do see why it is useful but still. I know there was issue with element (still are, rc.5 in npm) #1591, but it should be fixed in repo.

Reply:
Right, yeah, that must have been the error that was thrown.

Reply:
And as for the value of TypeScript, it's actually telling us a useful bit of information here:

If the element returned from `.get(0)` is `undefined`, then there will be an error with `cheerio.load()`.

So actually, a more robust version of the code would be this:

```ts

const $ul = $('ul').get(0);

if ($ul) {
  const $fromElement = cheerio.load($ul);

  if ($fromElement('ul > li').length !== 3) {
    throw new Error(
      'Expecting 3 elements when passing `cheerio.Element` to `load()`'
    );
  }
}

```

But since these are just tests, it doesn't matter too much that there is a cast here.